### ISSUE TITLE
list index out of range ——>xls_font = self.book.font_list[xf.font_index]       # Font object

### ISSUE THREAD

**Issue by admlke** [snoopyjc/xls2xlsx#11]
* xls2xlsx version: latest
* Python version: 3.7
* Operating System:

### Description
xls to xlsx
Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.
special excel conversion
### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```
File "C:\softWare\Python\Python37\lib\site-packages\xls2xlsx\xls2xlsx.py", line 240, in to_xlsx
    font, fill, border, alignment, number_format, protection = self.xls_style_to_xlsx(sheet.cell_xf_index(row, col))
  File "C:\softWare\Python\Python37\lib\site-packages\xls2xlsx\xls2xlsx.py", line 105, in xls_style_to_xlsx
    xls_font = self.book.font_list[xf.font_index]       # Font object
IndexError: list index out of range

**Comment by snoopyjc** [comment]
Can you share the input file?

**Comment by admlke** [comment]
> Can you share the input file?

sorry the data is private and cannot be provided, but it is normal to remove the data

**Comment by snoopyjc** [comment]
Work-around fixed in v0.2.0 - please try it and re-open if need be
